### ISSUE TITLE
why the loss function have no integral operation as in the paper? how to achieve IKL‘s I haha ?

### ISSUE THREAD

**Issue by Bigwode** [pkulwj1994/diff_instruct#2]

**Comment by pkulwj1994** [comment]
Hi there, thank you for liking the Diff-Instruct. The integral is implemented through stochastically sampling a random time (noise sigma) and then compute the loss. This follows the same spirit as used in diffusion models training.

**Comment by Bigwode** [comment]
thanks, it make sense but can you give some reference materials to demonstrate this operation?

**Comment by ShunLu91** [comment]
It may help that the reviewer [62Pb](https://openreview.net/forum?id=MLIs5iRq4w&noteId=o34XXNiXIe) asked the same question. The batch-average manner plays the role for the integral operation.
